src/content/docs/it/recipes.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Altre soluzioni
+description: Esempi guidati su come aggiungere nuove funzionalità al tuo progetto Astro.
+i18nReady: true
+---
+
+import RecipesNav from '~/components/RecipesNav.astro';
+
+Esempi guidati su come aggiungere nuove funzionalità al tuo progetto Astro.
+
+## Soluzioni ufficiali
+
+<RecipesNav />
+
+## Soluzioni dalla community
+
+Se hai una soluzione che vuoi condividere, consulta [le nostre linee guida per contribuire](https://github.com/withastro/docs/blob/main/contributor-guides/submitting-a-recipe.md)  
+
+- [Importare immagini utilizzando un nome file dinamico](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
```

Comment:
**question:** Since the original source of the recipe is in English, should we add a note (e.g. `<sup>EN</sup>` to let users know that the original recipe is not in Italian? Something similar to 

<img width="126" alt="image" src="https://github.com/withastro/docs/assets/11388820/c326b373-d665-44b5-8b0d-51cd2a40647d">

in https://docs.astro.build/it/getting-started/ ?

Other translations are not including this note (e.g. the Spanish one: https://docs.astro.build/es/recipes/ ), nevertheless we could add it, if it can be useful for Italian readers

Reply:
Those annotations are only used in the navbar and refer to pages that are present on the docs website. Adding them to external links would require a new component and it would probably impact the overall readability of the page.

It's not worth it in my opinion.